website/docs/docs/core/connect-data-platform/postgres-setup.md

```diff
@@ -93,3 +93,57 @@ If the database closes its connection while dbt is waiting for data, you may see
 
 If `dbt-postgres` encounters an operational error or timeout when opening a new connection, it will retry up to the number of times configured by `retries`. The default value is 1 retry. If set to 2+ retries, dbt will wait 1 second before retrying. If set to 0, dbt will not retry at all.
 
+
+### `psycopg2` vs `psycopg2-binary`
+
+`psycopg2-binary` is installed by default when installing `dbt-postgres`.
+This is ideal for development and testing workflows where performance is less of a concern.
+However, production environments will benefit from `psycopg2`, which is built from source for that particular operating system and archtecture.
+
+<VersionBlock firstVersion="1.8">
+
+To use `psycopg2`, simply uninstall `psycopg2-binary` and install the equivalent version of `psycopg2` after installing `dbt-postgres`:
+```bash
+pip install dbt-postgres
+if [[ $(pip show psycopg2-binary) ]]; then
+    PSYCOPG2_VERSION=$(pip show psycopg2-binary | grep Version | cut -d " " -f 2)
+    pip uninstall -y psycopg2-binary
+    pip install psycopg2==$PSYCOPG2_VERSION
+fi
+```
+
+</VersionBlock>
+
+<VersionBlock lastVersion="1.7">
+
+To use `psycopg2`, ensure that `DBT_PSYCOPG2_NAME=psycopg2` when installing `dbt-postgres`.
+The easiest way to do this is to set it inline while installing:
+```bash
+DBT_PSYCOPG2_NAME=psycopg2 pip install dbt-postgres
+```
+
+</VersionBlock>
+
+Installing `psycopg2` requires OS level dependencies, which is one of the reasons users opt for `psycopg2-binary` (the other reason being a much longer installation time).
```

Comment:
I like to group reasons together rather than using parenthetical. 
```suggestion
Some systems prefer `psycopg2-binary` since use of `psycopg2` requires OS level dependencies and a much longer installation time.
```

Reply:
I wouldn't say that the system prefers it, rather the user may prefer it on some systems. Though I agree this is a bit wordy; I'll try rewording it a bit.